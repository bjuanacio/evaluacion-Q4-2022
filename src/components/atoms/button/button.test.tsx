import { render } from "@testing-library/react";
import Button from "./button";

describe("button", () => {
  test("primary", () => {
    const { container } = render(<Button variant="primary" />);
    expect(container).toMatchSnapshot();
  });
  test("secondary", () => {
    const { container } = render(<Button variant="secondary" />);
    expect(container).toMatchSnapshot();
  });
});
