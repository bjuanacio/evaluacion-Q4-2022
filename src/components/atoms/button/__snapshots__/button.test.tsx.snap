// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button primary 1`] = `
<div>
  <button
    class="button button--primary"
  />
</div>
`;

exports[`button secondary 1`] = `
<div>
  <button
    class="button button--secondary"
  />
</div>
`;
